src/xo-app/dashboard/overview/index.js

```diff
@@ -381,9 +530,48 @@ export default class Overview extends Component {
           </Col>
         </Row>
       </Container>
-    ) : (
+    )
+  }
+}
+
+// ===================================================================
+
+@addSubscriptions({
+  resourceSets: subscribeResourceSets,
+  permissions: subscribePermissions,
+})
+@connectStore({
+  isAdmin,
+})
+export default class Overview extends Component {
+  render () {
+    const { props } = this
+    const showResourceSets = !isEmpty(props.resourceSets) && !props.isAdmin
+    const authorized = !isEmpty(props.permissions) || props.isAdmin
+
+    if (process.env.XOA_PLAN < 3) {
+      return (
+        <Container>
+          <Upgrade place='dashboard' available={3} />
+        </Container>
+      )
+    }
+
+    if (!authorized && !showResourceSets) {
+      return <h2>{_('notEnoughPermissionsError')}</h2>
```

Comment:
We usually put these messages in a `<em>`. We probably should have a generic error component for this anyway.